client/my-sites/patterns/components/pattern-library/index.tsx

```diff
@@ -186,7 +186,7 @@ export const PatternLibrary = ( {
 				searchTerm={ urlQuerySearchTerm }
 			/>
 
-			<DocumentHead title={ translate_not_yet( 'WordPress Patterns - Category' ) } />
+			<DocumentHead title={ translate_not_yet( `WordPress ${ category } Patterns ` ) } />
```

Comment:
I went with the previously recommended title here (see p1711382186987629-slack-C06ELHR6L9J), but I'm not sure if this format will always read the way we want, particularly after translation.

For example, "WordPress Form Patterns" reads well to me, but something like "WordPress About Patterns" feels a little off. Single quotes might help, i.e. "WordPress 'About' Patterns", but I'm not sure if we want it formatted that way or how that well that would work in terms of translation. Definitely open to input from reviewers 🤔

Reply:
We can't really localize a string like so, i.e. where different pieces are concatenated and which includes a variable ([documentation](https://github.com/Automattic/wp-calypso/blob/trunk/packages/i18n-calypso/README.md#strings-only)). I believe we will have to follow [the approach](https://github.com/Automattic/wp-calypso/blob/975b3c2a995d3f39bd8e1904d1b90c0276133116/client/my-sites/themes/use-theme-showcase-logged-out-seo-content.js#L29-L35) of the `Themes` page where we define a different title for each category. We will have to come up with different meta descriptions for SEO purposes anyway (I don't know if we also want to have a different H1 for each category though).

Reply:
And now that I'm swapping in `translate`, I'm hitting the exactly the error you were referencing. There are a couple of other spots where we also have variables in the translation strings that i'll tackle at the same time.

Thanks for the point to the `Themes` page, that's a good starting point to work around this 🙏

Reply:
Done - I've intentionally structured the object holding the `title` translations so we can easily add `descriptions` later as well.